.github/CONTRIBUTING.md

```diff
@@ -169,12 +169,32 @@ See [Contributing to documentation related to PowerShell](#contributing-to-docum
 
 #### Pull Request - Code Review
 
-* After a successful test pass,
-  the area maintainers will do a code review,
+* Roles of a PR: Author, Reviewer, and Assignee
+  * Reviewer and Assignee are two separate roles of a PR.
+    * A Reviewer can be anyone from the collaborators.
```

Comment:
Changed to `A Reviewer can be anyone who wants to contribute`.

> Probably also mention that PRs with Needs Review label needs a Reviewer

Do you mean the `Review - Needed` label? I'm afraid this label doesn't mean a reviewer needs to be assigned to a PR. I explicitly requested reviewers in #3170, but the `Review - Needed` label still got added to that PR.